terms.html

```diff
@@ -73,15 +70,15 @@
   <dt><dfn data-lt="holders|holder's|holders'">holder</dfn></dt>
   <dd>
 A role an <a>entity</a> might perform by possessing one or more
-<a>verifiable credentials</a> and generating <a>verifiable presentations</a> from them.
+<a>verifiable credentials</a> and creating <a>verifiable presentations</a> from them.
 A holder can be the <a>subject</a> of zero or more  <a>claims</a> within
 the <a>verifiable credentials</a> they are holding.
 Holders store their <a>credentials</a> in <a>credential repositories</a>.
   </dd>
   <dt><dfn class="lint-ignore"
            data-lt="identities|identity's">identity</dfn></dt>
   <dd>
```

Comment:
This description of `identity` seems more appropriate to `identifier`. My `identity` _is not_ a means of keeping track of me; my `identifier` *is*. I am not making a suggestion of how to change the description yet, but changes will be appropriate if line 79 is changed as noted.

We might — or might not — need to define both `identifier` and `identity`.....
```suggestion
  <dt><dfn class="lint-ignore"
           data-lt="identifiers|identifier's">identifier</dfn></dt>
  <dd>
```